src/css/profile/mobile/common/slider.less

```diff
@@ -46,17 +58,18 @@
 	}
 
 	&.ui-disabled {
-		opacity: 1;
 		.ui-slider-bar {
-			background-color: var(--slider-bg-disabled-color);
+			border-image: url(images/3_Controllers/tw_progress_bar_d.9.png) 5 fill;
+			opacity: 15%;
 
 			.ui-slider-value {
-				background-color: var(--slider-handler-disabled-color);
+				border-image: url(images/3_Controllers/tw_progress_bar_d.9.png) 5 fill;
 			}
 		}
 
 		.ui-slider-handler {
 			background-color: var(--slider-handler-disabled-color);
+			z-index: 9;
```

Comment:
Question) Is there any reasons to use the '9'?

Reply:
not exactly, I just needed to give it higher riority than other elements